### PR TITLE
*: Pass make race in the DDL test

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: ddl/ddl_db_test.go [hunk 1]
```diff
@@ -103,7 +106,12 @@ func (s *testDBSuite) testAddIndex(c *C) {
 	}
 
 	go func() {
-		s.mustExec(c, "create index c3_index on t1 (c3)")
+		se, err := tidb.CreateSession(s.store)
```
Comment (ngaut) [inline ddl/ddl_db_test.go]: Abstract to a new function. DRY.

**Comment by zimulala** [comment]
PTAL @ngaut @shenli @coocood

**Comment by ngaut** [comment]
LGTM

File: ddl/ddl_db_test.go [hunk 2]
```diff
@@ -59,6 +61,7 @@ func (s *testDBSuite) SetUpSuite(c *C) {
 	s.store, err = tidb.NewStore(uri)
 	c.Assert(err, IsNil)
 
+	localstore.MockRemoteStore = true
```
Comment (shenli) [inline ddl/ddl_db_test.go]: Why set this to true?

Comment (zimulala) [inline ddl/ddl_db_test.go]: Make set lease successfully. If local store the lease is 0 and can't be reset.

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**